README.md

```diff
@@ -178,29 +206,48 @@ The following sections provide instructions for these combinations:
   - `sudo apt update`
 - Install the AMD graphics driver and supporting firmware
   - `sudo apt install firmware-linux firmware-linux-nonfree libdrm-amdgpu1 xserver-xorg-video-amdgpu`
-- Install the OpenCL headers, OpenCL Installable Client driver, OpenCL lib, `git` and `go`
-  - `sudo apt install opencl-headers mesa-opencl-icd ocl-icd-libopencl1 git golang`
-- Help the loader find the OpenCL library by creating a symbolic link to it:
-  - `ln -s /usr/lib/x86_64-linux-gnu/libOpenCL.so.1 /usr/lib/libOpenCL.so`
+- Install the basic development tools `git` and `go`:
+  - `sudo apt install git golang`
 - Obtain the `gominer` source code
   - `git clone https://github.com/decred/gominer`
+- Jump to the appropriate section for either
+  [OpenCL](#opencl-on-debian) or [CUDA](#cuda-on-debian) 
+  depending on which GPU library you want to build `gominer` for.
+
+
+###### OpenCL on Debian
+
+- Install the OpenCL headers, OpenCL Installable Client driver, OpenCL lib
+  - `sudo apt install opencl-headers mesa-opencl-icd ocl-icd-libopencl1`
+- Help the loader find the OpenCL library by creating a symbolic link to it:
+  - `ln -s /usr/lib/x86_64-linux-gnu/libOpenCL.so.1 /usr/lib/libOpenCL.so`
 - Build `gominer`
   - `cd gominer`
   - `go build -tags opencl`
 - Test `gominer` detects your GPU(s)
   - `./gominer -l`
 - You may now [configure and run](#configuring-gominer) `gominer`
 
-#### CUDA Build Instructions (NVIDIA only)
 
-**Build instructions are not available yet.  They will be provided at a later
-date**.
+###### CUDA on Debian
```

Comment:
This section seems off in its placement.  It's a subsection of "AMD on Debian Bookworm".

Reply:
Do you think I should drop this (and leave CUDA only on the Ubuntu section) or rename the parent to just "Debian Bookwork"?

Reply:
Yeah, probably just making it Debian Bookworm and then a slight rearrangement to refactor out the common bits and then something like:

* Jump to the appropriate section for either [OpenCL with AMD](#link), [OpenCL with NVIDIA](#link), or [CUDA with NVIDIA](#link) depending on which type of GPU you have and the desired tookit